source/containers.tex

```diff
@@ -10480,13 +10480,17 @@
     // bit reference
     class @\libmember{reference}{vector<bool>}@ {
     public:
-      constexpr reference(const reference&) = default;
+      constexpr reference(const reference&) noexcept;
```

Comment:
```suggestion
      constexpr reference(const reference& x) noexcept;
```
There is also no `x` in the paper though. However, the paper does add an `x` in the synopsis of `bitset::reference`, and the whole point is harmonization, so I guess there ought to be an `x` here.